tensorflow_probability/python/stats/sample_stats.py

```diff
@@ -14,10 +14,20 @@
 # ============================================================================
 """Functions for computing statistics of samples."""
 
+JAX_MODE = False
+NUMPY_MODE = False
+
 # Dependency imports
 import numpy as np
 import tensorflow.compat.v2 as tf
 
+if NUMPY_MODE:
```

Comment:
Thanks for the review!

1. I don't feel comfortable rewriting `take_along_axis` as it would duplicate already existing logics, I feel like it would produce unnecessary maintenance burden.
2. What about [mapping](https://github.com/tensorflow/probability/pull/1600/commits/26f4f121a2ac4b2b14f15af38ebce944ba3f0fa8) `tensorflow.experimental.numpy` to `numpy` and `jax.numpy` backends?